umade-server/src/main/java/com/umade/utils/database/ProductsDAO.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package com.umade.utils.database;
+
+import com.umade.models.Product;
+import com.umade.utils.SQLDatabase;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class ProductsDAO extends SQLDatabase {
+    private String table;
+    
+    public ProductsDAO(Connection connection) {
+        super(connection);
+    }
+    
+    public ArrayList<Product> findPaidProduct(){
+        ArrayList<Product> prdList= new ArrayList<>();
+        
+        ResultSet rs = executeQueryPreparedStatement(
+                    "SELECT products.id, products.name, products.description, products.tags, products.image, "
+                    + "products.price, products.amount, products.sellBy, products.createdDate, products.updatedDate, products.isDeleted, users.name AS sellerName "
+                    + "FROM " + table + " "
+                    + "INNER JOIN cartItems ON products.id = cartItems.productId "
+                    + "INNER JOIN carts ON cartItems.cartId = carts.id "
+                    + "INNER JOIN payments ON payments.id = carts.paymentId "
+                    + "INNER JOIN users ON products.sellBy = users.id "
+                    + "WHERE payments.isPaid = 1"
+            );
+        try {
+            while (rs.next()) {     
+                Product p = new Product(
+                        rs.getInt("id"),
+                        rs.getNString("name"),
+                        rs.getNString("description"),
+                        rs.getNString("tags"),
+                        rs.getNString("image"),
+                        rs.getInt("price"),
+                        rs.getInt("amount"),
+                        rs.getInt("sellBy"),
+                        rs.getTimestamp("createdDate"),
+                        rs.getTimestamp("updatedDate"),
+                        rs.getBoolean("isDeleted"),
+                        rs.getNString("sellerName")
+                );
```

Comment:
Product doesn't have sellerName field
```suggestion
                Product p = new Product(
                        rs.getInt("id"),
                        rs.getNString("name"),
                        rs.getNString("description"),
                        rs.getNString("tags"),
                        rs.getNString("image"),
                        rs.getInt("price"),
                        rs.getInt("amount"),
                        rs.getInt("sellBy"),
                        rs.getTimestamp("createdDate"),
                        rs.getTimestamp("updatedDate"),
                        rs.getBoolean("isDeleted")
                );
```